Fix:
error: implicit declaration of function 'rmdir' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- sysdep.c.orig	2009-04-11 18:56:11.000000000 -0500
+++ sysdep.c	2020-09-29 09:41:32.000000000 -0500
@@ -20,6 +20,7 @@
 other tortious action, arising out of or in connection with the
 use or performance of this software.
 ****************************************************************/
+#include <unistd.h>
 #include "defs.h"
 #include "usignal.h"
 
